src/pspm_get_events.m

```diff
@@ -108,7 +115,7 @@
         baseline = v(end, 1);
         
         % we are interested in the delta -> remove "baseline offset"
-        values = data(round(mPos)) - baseline;
+        values = data_orig(round(mPos) - 3) - baseline;
```

Comment:
I am not really happy by this `-3` part. Maybe there is a better way to implement it?

Reply:
What this 3 is doing here ? Why do we need it ?